src/pages/en/guides/debugging.md

```diff
@@ -4,4 +4,35 @@ title: Debugging
 description: Debug in Astro using the Debug component
 ---
 
-Astro runs on the server and logs directly to your terminal, so it can be difficult to debug values from Astro. Astro’s built-in `<Debug>` component can help you inspect values inside your files on the clientside. Read more about the [built-in Debug Component](/en/reference/builtin-components#debug-).
+There are a few ways to debug your code with Astro. But it is important to remember that because **Astro runs on the server**, any code that is executed within the `---` Astro Front-matter is `console.log()` to the **terminal**.
```

Comment:
“frontmatter” seems to be all one word elsewhere. What do you think about mentioning the browser console explicitly as that’s the other place you might think or hope to see things logged.

```suggestion
There are a few ways to debug your code with Astro. But it is important to remember that because **Astro runs on the server**, any code that is executed within the Astro frontmatter will be logged to the **terminal**, not to the browser console.
```

Reply:
Haha, yeah, it was more fun to type than to read, for sure.

And I agree with both of you @aFuzzyBear  @delucis -- no image (12 reasons, maybe 15...) but since this is unfamiliar territory, I wasn't sure what might be expected in terms of preparing the user for the debugging output.

If I want to get super-technical, is *any* code executed in the front matter logged to the terminal, or just any console.log() output?

Reply:
1. Code executed in the frontmatter is just executed, `console.log` is an API for printing — a.k.a. logging — values. In the case of the frontmatter, it passes them to the terminal’s [`stdout`](https://en.wikipedia.org/wiki/Standard_streams#Standard_output_(stdout)). (In the browser the same API prints to the browser console, which I don’t think has the same concept of `stdout` as terminals.) Actually here’s the `console.log` definition from Node’s docs: “Prints to `stdout` with newline.”

2. I could back that, sure. The formatting does look slightly odd, so maybe just saying that it will render debugging information on the page is enough.

Reply:
So, "any code that is executed within the Astro frontmatter will be logged to the **terminal**" is not *technically* correct... that code is executed, but only stuff explicitly `console.log()`'d  is logged to the terminal, right? That's what I'm trying to verify.  :)

(That would have been my guess, but I do want to make sure here.)

Reply:
Ohhh, right, yes, you’ve got it spot on.

Reply:
OK, someone please put me out of my misery and make this both accurate, and a sensible sentence: @aFuzzyBear @delucis 

because **Astro runs on the server**, the output of `console.log()` written in Astro frontmatter will be logged to the **terminal**, not to the browser console.

Reply:
I think this is good! Slight tweak:

> …because Astro runs on the server, a `console.log()` in Astro frontmatter will output to the **terminal**, not the browser console.

Better/worse? I’m sure you’ll make the right call!